src/App.tsx

```diff
@@ -1,61 +1,102 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoInfo } from './components/TodoInfo';
+import { UserInfo } from './components/UserInfo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const [addTodos, setAddTodos] = useState(todosFromServer);
+
+  const handleTitleChange = (
+    setter: React.Dispatch<React.SetStateAction<string>>,
+    value: string,
+  ) => {
+    setter(value);
+    setHasTitleError(false);
+  };
+
+  const handleUserIdChange = (
+    setter: React.Dispatch<React.SetStateAction<number>>,
+    value: number,
+  ) => {
+    setter(value);
+    setHasUserIdError(false);
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!title) {
+      setHasTitleError(true);
+    } else {
+      setHasTitleError(false);
+    }
+
+    if (!userId) {
+      setHasUserIdError(true);
+    } else {
+      setHasUserIdError(false);
+    }
+
+    if (!title || !userId) {
```

Comment:
<img width="223" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/389c3c8c-190c-4955-adbe-5e3a0e8da173">
i can create the todo only with spaces, let's fix that